### ISSUE TITLE
Třídění podle Č a Š ve startovní listině po kategoriích

### ISSUE THREAD

**Issue by frantisekpasek** [fvacek/quickbox#298]
Jména začínající na Č a Š jsou zařazena na konci kategorie místo na místech kam podle abecedy patří.

![obrazek](https://user-images.githubusercontent.com/18633350/36644899-23a0cf16-1a61-11e8-99c1-e7b24e52e603.png)

**Comment by paukert** [comment]
Děje se pouze při použití SQLite, pro opravu by bylo potřeba asi vytvořit vlastní collation viz:
https://www.mimec.org/blog/qt-sqlite-and-locale-aware-collation
http://www.sqlite.org/c3ref/create_collation.html
https://forum.qt.io/topic/74954/qt-sqlite-plugin-building-case-insensitive-matching-of-unicode-characters-in-sqlite

**Comment by fvacek** [comment]
to asi nechame, jak to je, protoze jednoduche reseni neexistuje a postgres je reseni tem, kde by to opravdu vadilo, vyznamnejsi zavody se vetsinou poradaji na postgresu

**Comment by paukert** [comment]
Souhlasím, navíc se IMO v QE primárně podle jména řadí v minimálním počtu případů.

**Comment by otahirs** [comment]
Kdysi jsem se na to díval a bylo to az moc práce pro takovou maličkost `not
gonna implement`

**Comment by jmacura** [comment]
Bylo by fajn tuto "known limitation" aspoň zmínit v nějakém návodu/na wiki spolu s radou, jak se problému vyhnout.
